cmd/goal/defaultDevNetworkTemplate.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
I'm not sure we need both of these files. Also, the `NetworkName` seems to be wrong.

Under `Genesis` it should include:
*  `"LastPartKeyRound": 30000`
* `"RewardsPoolBalance": 0`

Also, I think there should just be a single node.

Everything except the `"DevMode": true` flag from [this template](https://github.com/algorand/sandbox/blob/master/images/algod/DevModeNetwork.json).

Reply:
What should the network name be? In your example it's devmodenet. What about not dev mode?

Reply:
How about "private network"?